shell/pages/c/_cluster/explorer/index.vue

```diff
@@ -764,20 +846,23 @@ export default {
   > I {
     text-align: center;
     padding: 5px 10px;
-    border-right: 1px solid var(--border);
   }
 
-  &.k8s-component-status-unhealthy {
+  &.unhealthy {
     border-color: var(--error-border);
 
     > I {
       color: var(--error)
     }
   }
 
-  &.k8s-component-status-healthy {
-    border-color: var(--border);
+  &.warning {
```

Comment:
As for the icon, we have a map for the colors in the states: https://github.com/rancher/dashboard/blob/master/shell/plugins/dashboard-store/resource-class.js#L400